crates/edr_napi/src/trace/mapped_inlined_internal_functions_heuristics.rs

```diff
@@ -0,0 +1,189 @@
+//!  This file includes Solidity tracing heuristics for solc starting with
+//! version  0.6.9.
+//!
+//!  This solc version introduced a significant change to how sourcemaps are
+//!  handled for inline yul/internal functions. These were mapped to the
+//!  unmapped/-1 file before, which lead to many unmapped reverts. Now, they are
+//!  mapped to the part of the Solidity source that lead to their inlining.
+//!
+//!  This change is a very positive change, as errors would point to the correct
+//!  line by default. The only problem is that we used to rely very heavily on
+//!  unmapped reverts to decide when our error detection heuristics were to be
+//!  run. In fact, this heuristics were first introduced because of unmapped
```

Comment:
```suggestion
//!  run. In fact, these heuristics were first introduced because of unmapped
```

Reply:
Thanks - I mostly copied it from the original file and didn't grammar/spell check it :see_no_evil: